src/utils/quote_report_utils.ts

```diff
@@ -82,4 +149,164 @@ export const quoteReportUtils = {
             });
         });
     },
+    publishQuoteReport(logOpts: ExtendedQuoteReportLogOptions, isFirmQuote: boolean, kafkaProducer: Producer): void {
+        const qr: ExtendedQuoteReport = logOpts.quoteReport;
+        let logBase: { [key: string]: string | boolean | number | undefined } = {
```

Comment:
I think we can drop the types here and allow typescript to do type inference, i.e.:
```
const qr = logOpts.quoteReprt;
let logBase = {
    ....
}
```

Reply:
Sure